src/main/java/services/UserService.java

```diff
@@ -1,7 +1,51 @@
 package services;
 
+import models.Ticket;
 import models.User;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import repository.DAOTicket;
+import repository.DAOUser;
 
+@Service
+@Transactional
```

Comment:
when you use this annotation with class level, then all methods will be performed under transactions. In some cases, for example when we have READ-only opeations from db, we do not need to do it with transaction. Better use this annotation with explicitly specific methods where transaction is required.